autoscheduler/scraper/management/commands/utils/scraper_utils.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+def get_all_terms() -> List[str]:
+    """ Generates all of the terms, from 2019 until now """
+
+    # Figure out the best way to do this
+    # Might want to read in the current year?
+    years = [2013 + i for i in range(7)]
+    semesters = [1, 2, 3]
+    locations = [1, 2, 3]
+
+    terms = []
+
+    for year in years:
+        for semester in semesters:
+            for location in locations:
+                terms.append(f"{year}{semester}{location}")
+
+    return terms

```

Comment:
```suggestion
    return [f"{year}{semester}{location}"
             for year, semester, location in product(years, semesters, locations)]
```

Reply:
ooooh see I didn't think want to do list comprehension cause I generally don't like having multiple loops but freaking frick this is clean, nice